Modal/Create Event

leong96 opened the pull request:
Reverted modal to original behavior - it will now show, but will always delete the administrator account. Eli is going to take a stab at this.

Create Event: Switched "lng" out for "lon" and returned those fields to being hidden.

leong96 committed:
Modal/Create Event

Reverted modal to original behavior - it will now show, but will always delete the administrator account. Eli is going to take a stab at this.

Create Event: Switched "lng" out for "lon" and returned those fields to being hidden.

justwheel reviewed (approved):
Since this is a revert, this is simple enough to me. I'm leaving an approval, but you can follow up with @ldidonato or @eguy006 if you also want them to take a look first too. :+1:

leong96 committed:
Lon / Lng

"lng" is required by the API. As such, all instances of "lon" have been replaced by "lng".

leong96 commented:
@jwflory @ldidonato @eguy006 Note an additional commit at the end; Lauren mentioned that "lng" is required for the API. As such, all mentions of "lon" have been replaced with "lng".

R-Eli-Levasseur reviewed (approved):
Looks alright to me

leong96 committed:
Transportation / Index

This commit removed Transportation from the appropriate strings, and makes it so that only activity categories are seen in index.php.

leong96 committed:
Merge branch 'master' into HLeong-Back-End
